nexus/temporal-payloads.md

```diff
@@ -0,0 +1,74 @@
+# Temporal Payloads over Nexus+HTTP
+
+## Motivation
+
+For ideal integration of Temporal applications with Nexus, Temporal users should be able to leverage Temporal Payloads
+and not have to concern themselves with Nexus implementation details.
+
+When a workflow completes, we'd like Temporal server to be able to deliver the completion result to any registered Nexus
+callbacks and the callback receiver (a Temporal server) to translate the completion to a Temporal Payload.
+
+Inherently, we need a fixed set of translation rules from Payloads to Nexus+HTTP and back. These rules should be
```

Comment:
We may want to also expose this at a later stage in the Temporal SDKs to allow using Nexus clients from application code but that's not required in the immediate future.